radixdlt/src/main/java/com/radixdlt/consensus/EpochManager.java

```diff
@@ -66,39 +65,54 @@ public EpochManager(
 	) {
 		this.proposalGenerator = Objects.requireNonNull(proposalGenerator);
 		this.mempool = Objects.requireNonNull(mempool);
-		this.networkSender = Objects.requireNonNull(networkSender);
+		this.sender = Objects.requireNonNull(sender);
 		this.safetyRules = Objects.requireNonNull(safetyRules);
 		this.pacemaker = Objects.requireNonNull(pacemaker);
-		this.pacemakerRx = Objects.requireNonNull(pacemakerRx);
 		this.vertexStore = Objects.requireNonNull(vertexStore);
 		this.pendingVotes = Objects.requireNonNull(pendingVotes);
 		this.proposerElectionFactory = Objects.requireNonNull(proposerElectionFactory);
 		this.selfKey = Objects.requireNonNull(selfKey);
 		this.counters = Objects.requireNonNull(counters);
 	}
 
-	public EventCoordinator start() {
-		return new EmptyEventCoordinator();
+	public BFTEventProcessor start() {
+		return new EmptyBFTEventProcessor();
 	}
 
-	public EventCoordinator nextEpoch(ValidatorSet validatorSet) {
+	public BFTEventProcessor nextEpoch(ValidatorSet validatorSet) {
 
 		ProposerElection proposerElection = proposerElectionFactory.create(validatorSet);
 		log.info("NEXT_EPOCH: ProposerElection: {}", proposerElection);
 
-		return new ValidatingEventCoordinator(
+		BFTEventReducer reducer = new BFTEventReducer(
```

Comment:
At some point soon, this really should change to using factories, rather than newing up concrete instances in this method.

This applies to BFTEventReducer and BFTEventPreprocessor.